pkg/scm/github.go

```diff
@@ -113,3 +114,29 @@ func (gh *Github) Setup() (con Context, err error) {
 	con.repoName = repoName
 	return
 }
+
+func (gh *Github) StarPluralGitHubRep() error {
+	ctx := context.Background()
+	starred, _, err := gh.Client.Activity.IsStarred(ctx, "pluralsh", "plural")
+	if err != nil {
+		return err
+	}
+	if !starred {
+		proceed := false
+		prompt := &survey.Confirm{
+			Message: "Would you like to Star us on the GitHub",
```

Comment:
let's change this message to `Would you like to show your support for the project with a github star?`